db/models.py

```diff
@@ -1 +1,43 @@
 from django.db import models
+
+
+class Race(models.Model):
+    races = (
+        ("Elf", "Elf"),
+        ("Dwarf", "Dwarf"),
+        ("Human", "Human"),
+        ("Ork", "Ork")
+    )
```

Comment:
There is no need to limit choices to these ones for this task, you should have only unique values for `name` without other constraints. Remove this code and redo migration to encounter that.